### PR TITLE
Translation polish

### LINKED ISSUE #690: Missing languages
Polish is translated but is not in the system by default, why?

https://docs.google.com/spreadsheets/d/1GzFPlVrqJBmUatfWft7v-vS_tfENGtAy0RHOv_5n3SU/edit#gid=0

https://github.com/GIScience/openrouteservice-app/#translations

> In this spreadsheet you can add a column for your language or check the already added words if they are correctly translated. Our service will be updated regularly with the latest language versions.

We checked and updated the translation so it's ok

### PR DESCRIPTION
**Pull request by takb**
### Pull Request Checklist
- [x] 1. I have [**rebased**](https://github.com/GIScience/openrouteservice/blob/master/CONTRIBUTE.md#pull-request-guidelines) the latest version of the master branch into my feature branch and all conflicts have been resolved.
- [x] 2. I have added information about the change/addition to functionality to the CHANGELOG.md file under the [Unreleased] heading.
- [ ] 3. I have documented my code using JDocs tags.
- [x] 4. I have removed unnecessary commented out code, imports and System.out.println statements.
- [ ] 5. I have written JUnit tests for any new methods/classes and ensured that they pass.
- [ ] 6. I have created API tests for any new functionality exposed to the API.
- [ ] 7. If changes/additions are made to the app.config file, I have added these to the app.config wiki page on github along with a short description of what it is for, and documented this in the Pull Request (below).
- [x] 8. I have built graphs with my code of the Heidelberg.osm.gz file and run the api-tests with all test passing
- [x] 9. I have referenced the Issue Number in the Pull Request (if the changes were from an issue).
- [ ] 10. For new features or changes involving building of graphs, I have tested on a larger dataset (at least Germany) and the graphs build without problems (i.e. no out-of-memory errors).
- [ ] 11. For new features or changes involving the graphbuilding process (i.e. changing encoders, updating the importer etc.), I have generated longer distance routes for the affected profiles with different options (avoid features, max weight etc.) and compared these with the routes of the same parameters and start/end points generated from the current live ORS. If there are differences then the reasoning for these **MUST** be documented in the pull request.
- [x] 12. I have written in the Pull Request information about the changes made including their intended usage and why the change was needed.

Fixes #690 .

### Information about the changes
Added language resource file for Polish.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Polish translation ([#690](https://github.com/GIScience/openrouteservice/issues/690))
 ### Fixed
+- Fixed fallback to dynamic routing methods if bearings parameter set ([#702](https://github.com/GIScience/openrouteservice/issues/702))
```
Comment (aoles) [inline CHANGELOG.md:10]: Shouldn't this be rather part of another PR?

File: openrouteservice/src/main/java/org/heigit/ors/routing/RouteSearchParameters.java [hunk 2]
```diff
@@ -503,6 +503,10 @@ public void setBearings(WayPointBearing[] bearings) {
         this.bearings = bearings;
     }
 
+    public boolean hasBearings() {
```
Comment (aoles) [inline openrouteservice/src/main/java/org/heigit/ors/routing/RouteSearchParameters.java]: Changes to this file are not related to this PR.

**Review by aoles — CHANGES_REQUESTED** [review]
Sleek, ORS now gives me instructions in my mother tongue :]

Please consider updating `APIEnumsTest` as well.

Cheers,
Andrzej

File: CHANGELOG.md (hunk 1 above)
Comment (takb) [reply CHANGELOG.md:10]: it is, I branched off the wrong place...

**Review by aoles — APPROVED** [review]
Perfect, thanks a lot for addressing my comments!
